### PR TITLE
Feature/parent child layout misplaced

### PR DESCRIPTION
**Pull request by BennetLeff**
This should fix issue #47 (Trello must have changed their layout over the years and caused some parent-child-hierarchy buttons to display in the wrong place).

### CODE REVIEW

**Comment by sdbrown2** [comment]
added a question, otherwise looks good to me :+1:

File: src/js/plugins/inheritance.js [hunk 1]
```diff
@@ -994,9 +1002,16 @@
 
     updateInheritanceListInOpenedCardView: function (card) {
       if (card) {
-        var _descParentElement = document.querySelector('[attr="desc"]'),
+        // We want to grab the section of a Card which is right above description.
+        // addingMethod is used to call prependElement in helpers.js.
```
Comment (SeanMcTex) [inline src/js/plugins/inheritance.js]: This may or may not be true, since the code may later change what's in `addingMethod` before it gets called.

Suggestion: `addingMethod is used to specify which helpers.js method to call to add the buttons`.

Comment (BennetLeff) [reply src/js/plugins/inheritance.js]: Great point

File: package.json [hunk 2]
```diff
@@ -1,12 +1,11 @@
 {
   "name": "handsome-trello",
-  "version": "0.8.1",
+  "version": "0.8.2",
```
Comment (SeanMcTex) [inline package.json:3]: While I don't think it's worth holding things up for, this isn't actually a part of the "parent child layout misplaced" issue, and therefore shouldn't be part of this branch. (It seems a little pedantic, I know, but it pays off when you have to roll back a feature or choose not to include it in a given release.) Normally, per gitflow, we'd make a "release/0.8.2" branch off `develop` when we're ready to do a release, make these sorts of changes there, and eventually merge that branch back into both `develop` and `master`.

File: package.json [hunk 3]
```diff
@@ -35,5 +34,8 @@
   "bugs": {
     "url": "https://github.com/handsomecode/trello-parent-child/issues"
   },
-  "homepage": "http://labs.handsome.is/trello/"
+  "homepage": "http://labs.handsome.is/trello/",
+  "dependencies": {
```
Comment (SeanMcTex) [inline package.json:38]: This also appears to be outside the scope of the actual work on this feature, so normally we'd put it on its own `feature/add-gulp` branch. Don't sweat it this time around, though.

**Review by SeanMcTex — APPROVED** [review]
Looks good to me. Merge it, Dano!

File: package.json (hunk 2 above)
Comment (BennetLeff) [reply package.json:3]: Good point. Should I revert this change?

File: package.json (hunk 3 above)
Comment (BennetLeff) [reply package.json:38]: This must have been one of the build tools or something similar. I don't actually recall making this change myself.
